webapp/platform/client/src/client4.ts

```diff
@@ -2033,9 +2033,9 @@ export default class Client4 {
         );
     };
 
-    getPostsSince = (channelId: string, since: number, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false) => {
+    getPostsSince = (channelId: string, since: number, page = 0, perPage = PER_PAGE_DEFAULT, fetchThreads = true, collapsedThreads = false, collapsedThreadsExtended = false) => {
```

Comment:
Why not put the new params at the end of the method? It's not optimal context-wise, but it will improve backward incompatibility since the order of the params is important.

Reply:
It is to keep the signature similar to all other functions.